packages/pg-batch-query/bench.ts

```diff
@@ -0,0 +1,79 @@
+import pg from 'pg'
+import BatchQuery from './dist'
```

Comment:
question: is it a bit strange to require the dist folder here? Why not require from `./src`?

Reply:
Good catch. It seems IDE auto-imported it from `dist` . Will fix.